src/instance/InstanceAvailiabilityOptions.ts

```diff
@@ -0,0 +1,16 @@
+import {ApiProperty} from '@nestjs/swagger';
+import {IsNotEmpty, IsUUID} from 'class-validator';
+import {AvailabilityType} from '@wireapp/core/dist/broadcast';
+import {Availability} from '@wireapp/protocol-messaging';
+
+export class InstanceAvailiabilityOptions {
+  @ApiProperty({
+    enum: [Availability.Type.AVAILABLE, Availability.Type.AWAY, Availability.Type.BUSY, Availability.Type.NONE],
+  })
+  @IsNotEmpty()
```

Comment:
Can this be replaced with  `IsEnum`? -> https://github.com/typestack/class-validator#manual-validation